### DISCUSSION TITLE
Suggestion - Displaying carbon emission on the map

### DISCUSSION THREAD

**Discussion by tneme-lC** [electricitymaps/electricitymaps-contrib#4647] [Ideas]
Hello,
The current display is showing the carbon intensity in g eq CO2/kWh. This indicator is indeed very interesting to understand whether a country's own production is green or not. 

I think it would be very interesting to also be able to display the carbon emission in g eq CO2/min. This would allow to understand which are the most polluting countries. I know that those data are available in the "carbon emission"  tab, I would just suggest to display them on the map.

Many thanks for reading my suggestion, and I'm looking forward to discuss about it.

**Comment by VIKTORVAV99** [comment]
I think it's a good suggestion and something we should take a closer look at.
Perhaps when we take a crack at #4428 we can implement this as well.
@Kongkille maybe the team has some additional input regarding this?
